libs/platform/src/lib/components/form/select/select.component.spec.ts

```diff
@@ -1,6 +1,6 @@
 import { async, ComponentFixture, TestBed, fakeAsync, tick } from '@angular/core/testing';
 
-import { SelectPlatformComponent } from './select.component';
+import { PlatformSelectComponent } from './select.component';
```

Comment:
We should remove and comment out platform implemenation and submit this as separate PR. It should not be part of this